zone/zoning.cpp

```diff
@@ -189,41 +199,54 @@ void Client::Handle_OP_ZoneChange(const EQApplicationPacket *app) {
 
 	//handle circumvention of zone restrictions
 	//we need the value when creating the outgoing packet as well.
-	uint8 ignorerestrictions = zonesummon_ignorerestrictions;
+	uint8 ignore_restrictions = zonesummon_ignorerestrictions;
 	zonesummon_ignorerestrictions = 0;
 
-	float dest_x=0, dest_y=0, dest_z=0, dest_h;
-	dest_h = GetHeading();
+	float target_x = 0, target_y = 0, target_z = 0, target_heading = 0;
 	switch(zone_mode) {
 	case EvacToSafeCoords:
 	case ZoneToSafeCoords:
-		LogDebug("Zoning [{}] to safe coords ([{}],[{}],[{}]) in [{}] ([{}])", GetName(), safe_x, safe_y, safe_z, target_zone_name, target_zone_id);
-		dest_x = safe_x;
-		dest_y = safe_y;
-		dest_z = safe_z;
+		LogDebug(
+			"Zoning [{}] to safe coords ([{}], [{}], [{}]) in [{}] ([{}])",
```

Comment:
Missing a format specifier `[{}]` for the new `safe_heading` arg

Reply:
Fixed.